projects/utils-ogc/src/lib/wms/wms.service.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from '@angular/core';
```

Comment:
If you use `@angular/core` in `@dlr-eoc/utils-ogc` you have to add it to the package.json `peerDependencies`.

And maybe then adjust the documentation 
> ...to make all services in `@dlr-eoc/services-ogc` independent of angular...
> https://github.com/dlr-eoc/ukis-frontend-libraries/blob/v7.3.0-next.2/projects/utils-ogc/README.md#dlr-eocutils-ogc

Or you try to do it like you have done it for the [WpsClient](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/v7.3.0-next.2/projects/utils-ogc/src/lib/wps/wpsclient.ts)

But at least add `@angular/core` to the [devDependencies](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/v7.3.0-next.2/projects/utils-ogc/package.json#L14) because it is used in the tests.